### ISSUE TITLE
[CS 1.6] Lower ping with high fps?

### ISSUE THREAD

**Issue by djdallmann** [ValveSoftware/halflife#3034]
While playing online I get latency between 40-50ms @ 100fps but with 400fps I get 30-40 ms.  Curious if anyone has insights on this behavior and can also reproduce because that's a decent reduction in latency. To note I'm on Win10 20H2.

![image](https://user-images.githubusercontent.com/7385812/107321895-04d35080-6a69-11eb-8604-535aa61a37d3.png)
![image](https://user-images.githubusercontent.com/7385812/107321973-2af8f080-6a69-11eb-81c8-7a0183b1eceb.png)

**Comment by Maxi605** [comment]
Did you modify the "rate" command along the fps?

**Comment by djdallmann** [comment]
Nope, rate, updaterate, cmdrate remained the same. 30000, 100 , 105

**Comment by BlackShadow** [comment]
Yep this happens if you have high FPS. I believe this happens because in higher framerate client receive packets faster from server? I maybe completely incorrect, it's just a guess. It doesn't have anything with rate.

**Comment by APGRoboCop** [comment]
400 FPS is a lot but not quite necessary. Having the FPS set near 120 would be just enough for playing CS 1.6, but of course, it really depends on your ISP's broadband quality as well as the DNS.
